src/test/java/com/coherentsolutions/training/auto/API/pashkovskaya/tests/DummyTests.java

```diff
@@ -0,0 +1,71 @@
+package com.coherentsolutions.training.auto.API.pashkovskaya.tests;
+
+import com.coherentsolutions.training.auto.API.pashkovskaya.base.BaseTest;
+import com.coherentsolutions.training.auto.API.pashkovskaya.model.Employee;
+import io.restassured.RestAssured;
+import org.hamcrest.Matchers;
+import org.testng.annotations.Test;
+
+public class DummyTests extends BaseTest {
+    public static final String SUCCESS_MESSAGE_TEXT_EMPLOYEE_CREATING = "Successfully! Record has been added.";
+    public static final String SUCCESS_MESSAGE_TEXT_EMPLOYEE_DELETING = "Successfully! Record has been deleted";
+    public static final String SUCCESS_MESSAGE_TEXT_GETTING_EMPLOYEE = "Successfully! Record has been fetched.";
+    public static final String SUCCESS_MESSAGE_TEXT_GETTING_EMPLOYEES = "Successfully! All records has been fetched.";
+    public static final String SUCCESS_MESSAGE_TEXT_EMPLOYEE_UPDATING = "Successfully! Record has been updated.";
+
+    @Test
+    public void TestCreatingEmployee() {
+        Employee employee = new Employee("Vika", "10000", "27");
+
+        RestAssured
+                .given()
+                .when()
+                    .body(employee)
+                    .post("/create")
+                .then()
+                    .body("message", Matchers.equalTo(SUCCESS_MESSAGE_TEXT_EMPLOYEE_CREATING));
+    }
+
+    @Test
+    public void TestDeletingEmployee() {
+        RestAssured
+                .given()
+                .when()
+                .delete("/delete/1")
```

Comment:
are you sure that you'll always have record with id equal 1?
better to 
1) create employee
2) get employee(s)
3) delete employee
then your test will be stable